tests/ui/conftest.py

```diff
@@ -100,12 +101,17 @@ def user(
     }
     assert requests.codes.created == response.status_code
     user.update(response.json())
-    print(user)
     return user
 
 
 @pytest.fixture(scope='function')
-def our_live_server(request):
+def live_server(request):
+    """
+        This fixture overrides the live_server fixture provided by
+        pytest_django. live_server allows us to create a running version of the
+        addons django application within pytest for testing.
```

Comment:
I can answer this a bit more… apparently @jrbenny35 rewrote the git history so my commit got lost, this is what I wrote:

```
    Hacky but kinda reasonable solution to our liveserver jwt problem.

    Now off to a explanation.

    From what I found out was that the `live_server` fixture (in our setup,
    couldn't reproduce in a fresh project) apparently starts up the
    LiveServerThread way too early before pytest-django configures the
    settings correctly.

    That resulted in the LiveServerThread querying the 'default' database
    which was different from what the other fixtures and tests were using
    which resulted in the problem that the just created api keys could not
    be found in the api methods in the live-server.

    I worked around that by implementing the live_server fixture ourselfs
    and make it function-scoped so that it now runs in a proper
    database-transaction.

    This is a HACK and I'll work on a more permanent solution but for now it
    should be enough to continue working on porting tests...

    Also investigating if there are any problems in pytest-django directly.
```